src/Sparrow.Server/Platform/Pal.cs

```diff
@@ -80,17 +80,17 @@ static Pal()
 
                 rc = rvn_get_system_information(out SysInfo, out errorCode);
             }
-            catch (IncorrectDllException)
-            {
-                throw;
-            }
             catch (Exception ex)
             {
-                var ErrString =
-                    "'Microsoft Visual C++ 2015 Redistributable Package' (or newer). It can be downloaded from https://support.microsoft.com/en-us/help/2977003/the-latest-supported-visual-c-downloads";
-                throw new IncorrectDllException(
-                    $"{LIBRVNPAL} version might be invalid or not usable on current platform. Initialization error could also be caused by missing {ErrString}",
-                    ex);
+                var platformStr = $"Arch:{RuntimeInformation.OSArchitecture}, OSDesc:{RuntimeInformation.OSDescription}";
```

Comment:
Missing space after `Arch:` and `OSDesc:`. Do we really need that? It produces longer exception method and the assumption here is what? That the user does not know what OS he is running on? Same info is available in welcome message.

Reply:
I would suggest to remove it so the message would be more clear.

Reply:
This is for us so if we get this message from a client we don't need to do a ping pong with the client for this information

Reply:
So it should be at the end of the message. It hides the main part.

Reply:
O.k.